.github/workflows/python-ci.yml

```diff
@@ -1,6 +1,6 @@
 name: Python CI
 on:
-  pull_request_target:
+  pull_request:
```

Comment:
now forked repo PR CI runs will get a github access token that has read-only access and no secrets access.

this means for now that the integration tests will fail for forked PRs. We'll come up with a workflow where we can manually trigger integration test runs for external contributors.